packages/monocdk-experiment/build-tools/gen.ts

```diff
@@ -228,7 +228,7 @@ async function transformPackage(
 
     await fs.writeFile(
       path.resolve(LIB_ROOT, '..', `${library.shortName}.ts`),
-      `export * from './lib/${library.shortName}';\n`,
+      `import { ${library.namespace} } from './lib';\nexport = ${library.namespace};\n`,
```

Comment:
Just curious, why this change? I'm not super familiar with the `export = ` syntax but according to the TS docs
>When exporting a module using export =, TypeScript-specific import module = require("module") must be used to import the module.

https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require

Reply:
I basically wanted to get rid of the `index.ts` that was previously introduced *only* to back this particular entry point. But I reckon it is possible to achieve the same by using `export { namespace } from './lib';`.